templates/dmarc_reports/index.html.twig

```diff
@@ -8,74 +8,87 @@
+                                      {% if report.begintime|date("Y-m-d") != report.endtime|date("Y-m-d") %}
+                                        <span class="d-none d-lg-inline">
+                                          .. 
+                                          {{ report.endtime|date("Y-m-d H:i:s")|replace({ '00:00:00': '', '23:59:59': '' }) }}
+                                        </span>
+                                      {% endif %}
+                                    </div>
+                                  </td>
+                                  <td class="d-none d-lg-table-cell">{{ report.dmarc_records.count }}</td>
+                                  <td>{{ report.domain.fqdn }}</td>
+                                  <td class="d-none d-md-table-cell">{{ report.organisation }}</td>
+                                  <td class="d-none d-lg-table-cell" title="{{ report.externalid }}">{{ report.externalid|slice(0,8) }}...</td>
+                                  <td class="text-nowrap">
+                                    {% set dkim_pass = true %}
+                                    {% set spf_pass = true %}
+                                    {% for record in report.dmarc_records %}
+                                        {% if record.policydkim != "pass" %}
+                                          {% set dkim_pass = false %}
+                                        {% endif %}
+                                        {% if record.policyspf != "pass" %}
+                                          {% set spf_pass = false %}
+                                        {% endif %}
+                                    {% endfor %}
+                                    <span class="badge badge-pill text-bg-{% if spf_pass %}success{% else %}danger{% endif %}">SPF</span>
+                                    <span class="badge badge-pill text-bg-{% if dkim_pass %}success{% else %}danger{% endif %}">DKIM</span>
+                                  </td>
+                                  <td class="text-nowrap">
+                                    <a href="{{ path('app_dmarc_reports_report', {report: report.id}) }}" class="link-primary px-2" title="View details">
+                                      <!-- Bootstrap icon 'eye-fill' -->
+                                      <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-eye-fill" viewBox="0 0 16 16">
+                                        <path d="M10.5 8a2.5 2.5 0 1 1-5 0 2.5 2.5 0 0 1 5 0"/>
+                                        <path d="M0 8s3-5.5 8-5.5S16 8 16 8s-3 5.5-8 5.5S0 8 0 8m8 3.5a3.5 3.5 0 1 0 0-7 3.5 3.5 0 0 0 0 7"/>
+                                      </svg></a>
+                                    <a href="{{ path('app_dmarc_reports_delete', {report: report.id}) }}" class="link-danger" title="Delete entry ..." onclick="return confirm('{{'Are you sure? this can not be undone'|trans}}');">
```

Comment:
please remove ... in delete entry

Reply:
I respectfully disagree. This is basic HIG 101. 
"..." on a button or menu item indicates there will be a confirmation prompt or other dialog after clicking, which there is.
So - IMHO - the "..." belongs here, and probably in other places as well.

Reply:
i've never come across this ever.
can you provide me with some sources to this?

Reply:
> The crucial factor is whether the menu option requires additional information (input or a selection) before it carries out the operation. So Help-About doesn't require an ellipsis

this is just an conformation dialog. No additional information is needed.